### ISSUE TITLE
Not able to embedded video or insert video using Insert Video Button

### ISSUE THREAD

**Issue by Cokezaza** [fraywing/textAngular#1080]
Hi,

I'm tried to insert youtube video using insert video button, the video got converted to image instead. So I tried to embedded using html mode but I'm not able to do so. I tried on your demo as well but it isn't working too. 

Is it a bug?

**Comment by pbassut** [comment]
Nope, if you look at the rendered version it shows a video. The reason a image is added is because inside TA a playable video wouldn't be very good performance.

Enviado do meu iPhone
